presentation/src/main/java/com/example/myapplication/MainActivity.kt

```diff
@@ -3,44 +3,89 @@ package com.example.myapplication
 import android.os.Bundle
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
 import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.Surface
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
 import com.example.myapplication.ui.theme.MyApplicationTheme
+import com.example.myapplication.ui.theme.Pink
+import com.example.myapplication.ui.theme.White
 
 class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContent {
             MyApplicationTheme {
-                // A surface container using the 'background' color from the theme
                 Surface(
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colorScheme.background
                 ) {
-                    Greeting("Android")
+                    TestFrame()
                 }
             }
         }
     }
 }
 
 @Composable
-fun Greeting(name: String, modifier: Modifier = Modifier) {
-    Text(
-        text = "0일차 과제~!",
-        modifier = modifier
-    )
+fun TestFrame() {
+    Column(
+        modifier = Modifier
+            .background(White)
+            .fillMaxSize()
+            .padding(start = 20.dp, end = 20.dp, top = 35.dp),
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        Spacer(
```

Comment:
```suggestion
        Box(
        ...
        )
```
`Spacer()` 컴포저블 함수보단 해당 UI에서는 `Box()` 함수가 조금 더 어울릴것 같아요! 
`Spacer()`는 주로 공간을 만들때 사용되는 녀석이긴 하지만, 만일 영역 안에 다른 컴포저블 함수를 필요로 한다면 `Box()` 함수가 적절한 컴포저블 함수가 될 수 있을것 같습니다. :)